src/GafferScene/Transform.cpp

```diff
@@ -98,20 +113,67 @@ bool Transform::processesTransform() const
 
 void Transform::hashProcessedTransform( const ScenePath &path, const Gaffer::Context *context, IECore::MurmurHash &h ) const
 {
-	spacePlug()->hash( h );
+	const Space space = static_cast<Space>( spacePlug()->getValue() );
+	const Mode mode = static_cast<Mode>( modePlug()->getValue() );
+	h.append( space );
+	h.append( mode );
 	transformPlug()->hash( h );
+
+	if( space == World )
+	{
+		ScenePath parentPath = path;
+		parentPath.pop_back();
+		h.append( inPlug()->fullTransformHash( parentPath ) );
```

Comment:
To resolve the Absolute World across ... case, I changed this line to:

h.append( outPlug()->fullTransformHash( parentPath ) );